api/envoy/config/route/v3/route_components.proto

```diff
@@ -745,6 +745,9 @@ message RouteAction {
     message ConnectConfig {
       // If present, the proxy protocol header will be prepended to the CONNECT payload sent upstream.
       core.v3.ProxyProtocolConfig proxy_protocol_config = 1;
+
+      // If set, the TCP proxy will expect POST payload as raw TCP.
+      bool use_post = 2;
```

Comment:
update the comment below for connect_config, because it is not only about `CONNECT` request.